### DISCUSSION TITLE
FIlter Identities

### DISCUSSION THREAD

**Discussion by FerdinaKusumah** [ory/kratos#2615] [Q&A] [answered]
Is there any api or function that i can filter or search for specified identities ?

let say i want to filter for some email only, or based on group in properties ?

Thank you

**Comment by kmherrmann** [comment]
We have an open PR and discussions in #2442 - would you mind describing your use case there?

**Comment by vinckr** [comment, marked as answer]
Hello @FerdinaKusumah 
A bit late to let you know but this has been implemented: https://changelog.ory.sh/announcements/an-api-to-fetch-identities-by-email-or-username
